integration/update_compat_test.go

```diff
@@ -139,6 +143,10 @@ func testUpdateCompat(t *testing.T, testCases map[string]updateCompatTestCase) {
 				})
 			}
 
+			if skippedAllTigrisSubtests {
```

Comment:
If all the subtests are skipped, `tc.resultType` will not match because in case of Tigris the result will be empty, but expect result is "non-empty". As the result, the main test will fail even though all the subtests are skipped. Maybe there is a better way to skip All Tigris subtests in such cases, @chilagrow did you have anything similar while moving tests to compat tests?

Reply:
Thanks for the explanation. I now see the problem you are solving. I see you want `non-empty` check to be skipped for tigris.
Tigris does not allow schema change so renaming document key is not something that would be supported by tigris if I understand correctly. So I would skip all $rename tests for tigris :hugs:

Reply:
Let's do this https://github.com/FerretDB/FerretDB/pull/1753/files#r1066737442

Reply:
Technically we still support `$rename` for tigris, and it will work in some test cases that are here, so I don't know why would we remove tests for it